src/views/Layout.jsx

```diff
@@ -20,7 +20,7 @@ import './Layout.css';
 
 export function Layout() {
 	return (
-		<>
+		<Paper sx={{ minHeight: '100dvh' }}>
```

Comment:
We have to wrap the app in `Paper` to get the background colors to properly respect user preferences. As a bonus, we can now ensure it is always as tall as the viewpoert.